### PR TITLE
docs: add link to install a Medusa server

### PR DESCRIPTION
**Pull request by bytrangle**
In this article on [configuring a Medusa server](https://docs.medusajs.com/usage/configurations), it mentions that readers should have a Medusa server installed at this point.

However, it doesn't include the link to instruction for installing a Medusa server. The article right before this article is about setting up a development environment and didn't mention installing Medusa server either.

So I thought it would be clearer to add a link to the part on [installing Medusa server](https://docs.medusajs.com/quickstart/quick-start/#create-a-medusa-server).

### CODE REVIEW

File: docs/content/usage/configurations.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ In this document, you’ll learn what configurations you can add to your Medusa
 
 ## Prerequisites
 
-This document assumes you already followed along with the [“Set up your development environment” documentation](../tutorial/0-set-up-your-development-environment.mdx) and have installed a Medusa server.
+This document assumes you already followed along with the [“Set up your development environment” documentation](../tutorial/0-set-up-your-development-environment.mdx) and have [installed a Medusa server](../quickstart/quick-start-docker.md#create-a-medusa-server).
```
Comment (shahednasser) [inline docs/content/usage/configurations.md:7]:
```suggestion
This document assumes you already followed along with the [“Set up your development environment” documentation](../tutorial/0-set-up-your-development-environment.mdx) and have [installed a Medusa server](../quickstart/quick-start.md#create-a-medusa-server).
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just a small change below 🙂

File: docs/content/usage/configurations.md (hunk 1 above)
Comment (bytrangle) [reply docs/content/usage/configurations.md:7]: @shahednasser Thank you for the suggestion. So it should be Quick Start Docker, not the Quick Start with a barebone server?

And do you have a preferred commit message convention?

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Awesome thanks! @bytrangle

**Pull request merged**